HelloMixpanel/HelloMixpanelTests/HelloMixpanelTests.m

```diff
@@ -904,18 +861,18 @@ - (void)testPeopleClearCharges
     XCTAssertEqualObjects(r[@"$set"][@"$transactions"], @[]);
 }
 
-- (void)testDropEvents
-{
-    for (NSInteger i = 0; i < 5005; i++) {
-        [self.mixpanel track:@"rapid_event" properties:@{@"i": @(i)}];
-    }
-    [self waitForSerialQueue];
-    XCTAssertTrue([self.mixpanel.eventsQueue count] == 5000);
-    NSDictionary *e = self.mixpanel.eventsQueue[0];
-    XCTAssertEqualObjects(e[@"properties"][@"i"], @(5));
-    e = [self.mixpanel.eventsQueue lastObject];
-    XCTAssertEqualObjects(e[@"properties"][@"i"], @(5004));
-}
+//- (void)testDropEvents
```

Comment:
I thought you were bringing this back. Or at least remove it entirely from the file :)